heudiconv/convert.py

```diff
@@ -322,9 +322,8 @@ def convert(items, converter, scaninfo_suffix, custom_callable, with_prov,
                     )
 
         if len(bids_outfiles) > 1:
-            lgr.warning("For now not embedding BIDS and info generated "
```

Comment:
this warning is still in effect (somewhat).  We are not embedding "additional" metadata from DICOMs